index.html

```diff
@@ -467,6 +467,29 @@ <h3>SigLens vs InfluxDB/Parquet/Arrow</h3>
     </div>
   </section>
 
+  <section class="my-4 pb-5">
```

Comment:
where are we saving this information?

Reply:
We are saving it in the Airtable database.